app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -62,8 +55,8 @@ export const NOTES_FORM_INITIAL_FORM_VALUES = {
 export const NOTES_FORM_VALIDATION_SCHEMA = yup.object().shape({
   title: yup.string().required("Title is required"),
   description: yup.string().required("Description is required"),
-  contactList: yup.string().oneOf(CONTACTS_LIST),
-  tagList: yup.string().oneOf(TAGS_LIST),
+  contactList: yup.string().oneOf(CONTACTS),
+  tagList: yup.string().oneOf(TAGS),
```

Comment:
This validation doesn't seem right to me. Please go through it once again and see if there are any issues and get back to me.

Reply:
I have fixed the validation schema.